examples/example_vs.cpp

```diff
@@ -13,12 +13,11 @@ struct ExampleMiddleware
 {
     std::string message;
 
-    ExampleMiddleware() 
+    ExampleMiddleware() : message(std::string("foo"))
```

Comment:
again could you remove the std::string portion

Reply:
yes removed message was already declared as std::string so there is no need to typecaste to std::string again